packages/vuetify/src/components/VOverlay/positionStrategies.ts

```diff
@@ -208,10 +208,15 @@ function connectedPositionStrategy (data: PositionStrategyData, props: StrategyP
 
     const canFill = doesOverlap.value || ['center', 'top', 'bottom'].includes(anchor.side)
 
+    const maxWidthFromProps: number = !props.maxWidth ? Number.MAX_SAFE_INTEGER // use this constant to indicate no max width limit
```

Comment:
Is it necessary to use this? I don't see a specific benefit to it.

Reply:
Without this, when `maxWidth` is not set in props, `maxWidthFromProps` will be `undefined` which can cause `Math.min(undefined, 1)` to return `NaN` in line 216 & 217.

Reply:
Probably better to allow `undefined` here and do Falsy conditional check in 216 & 217.

Leave with me and I will optimise